data/src/error_analysis/data/cg.tsv

```diff
@@ -0,0 +1,358 @@
+	.
```

Comment:
This looks like more non-WikiPron Bengali, and also doesn't belong here.

Reply:
fixed